starboard/nplb/string_format_test.cc

```diff
@@ -25,7 +25,7 @@ namespace {
 int Format(char* out_buffer, size_t buffer_size, const char* format, ...) {
   va_list arguments;
   va_start(arguments, format);
-  int result = SbStringFormat(out_buffer, buffer_size, format, arguments);
```

Comment:
for the NPLB tests let's keep the original code but just guard the whole test with #if SB_API_VERSION < 16

Reply:
Please bring the check at the top of the file and don't change the symbol.

For NPLB we don't change the API we just turn off the whole test for SB_API_VERSION < 16.